NineChronicles.Headless/GraphTypes/States/ArenaRecordType.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Types;
+using Nekoyume.Model.State;
+
+namespace NineChronicles.Headless.GraphTypes.States
+{
+    public class ArenaRecordType : ObjectGraphType<ArenaInfo.Record>
+    {
+        public ArenaRecordType()
+        {
+            Field<IntGraphType>(
+                nameof(ArenaInfo.Record.Win),
+                resolve: context => context.Source.Win);
+            Field<IntGraphType>(
+                nameof(ArenaInfo.Record.Lose),
+                resolve: context => context.Source.Lose);
+            Field<IntGraphType>(
+                nameof(ArenaInfo.Record.Draw),
+                resolve: context => context.Source.Draw);
```

Comment:
The fields seem not nullable though it can be 0.

https://github.com/planetarium/lib9c/blob/a5339c0de85ec6cd9523e355f97096487eae1acf/Lib9c/Model/State/WeeklyArenaState.cs#L317-L319

```suggestion
            Field<NonNullGraphType<IntGraphType>>(
                nameof(ArenaInfo.Record.Win),
                resolve: context => context.Source.Win);
            Field<NonNullGraphType<IntGraphType>>(
                nameof(ArenaInfo.Record.Lose),
                resolve: context => context.Source.Lose);
            Field<NonNullGraphType<IntGraphType>>(
                nameof(ArenaInfo.Record.Draw),
                resolve: context => context.Source.Draw);
```